cv32e40p/tests/programs/corev-dv/corev_rand_pulp_hwloop_debug/corev-dv.yaml

```diff
@@ -0,0 +1,24 @@
+name: corev_rand_pulp_hwloop_debug
```

Comment:
Since we are here anyway, please add a copyright and SPDX header to all new files:
```
# Copyright 2023 Dolphin Design
# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Reply:
Added copyright and SPDX header for all these new files.